c10/core/DeviceType.cpp

```diff
@@ -131,6 +131,9 @@ std::string get_privateuse1_backend(bool lower_case) {
   // set, and will never be written to.
   auto backend_name =
       name_registered ? privateuse1_backend_name : "privateuseone";
+  auto op_case = lower_case ? ::tolower : ::toupper;
```

Comment:
The parameter "lower_case" probably doesn't make sense.

Option 1: Just remove the parameter
Option 2: Keep the parameter, but you may want to add it to the [Python API](https://github.com/pytorch/pytorch/blob/751c744ad0f3c22c544d9821b88d8ab4a0676783/torch/csrc/Module.cpp#L639).

Reply:
yes, `lower_case` is `true` by default, but it makes sense for other device types, so i think it should be kept. so should we chooice option 2?

Reply:
We need this parameter for the function above that maps device types to their names (not sure how we use it there, for some dispatcher key stuff I think).
I don't think we need to expose this to the python API unless we need it though.